src/main/kotlin/app/widgets/recordButton/Record.kt

```diff
@@ -0,0 +1,3 @@
+package app.widgets.recordButton
+
+data class Record(val audio: String, val image : String, val recording: Boolean)
```

Comment:
RecordState perhaps?

Reply:
Actually not using this file.  I'll remove it